neo4j/driver_examples_test.go

```diff
@@ -138,6 +143,32 @@ func ExampleDriver_verifyAuthenticationDriverLevel() {
 	handleError(err)
 }
 
+func ExampleConfig_tlsSelfSignedCertificates() {
```

Comment:
```suggestion
func ExampleDriver_tlsSelfSignedCertificates() {
```
Config does not live in this package. So either rename the example like above or move it into the `config` package to be displayed next to `config.Config` (instead of next to `neo4j.Driver`).